src/features/aptosSignIn.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Signature } from '@aptos-labs/ts-sdk'
+import { UserResponse } from '../misc'
+import { AccountInfo } from '../AccountInfo'
+
+/** Version of the feature. */
+export type AptosSignInVersion = '1.0.0'
```

Comment:
How about using `1.0`?
I don't think the patch is adding any value to feature definition.

Maybe the wallet implementing the feature could decide to bump the patch, but also not sure if that would matter

Reply:
as mentioned on slack, how about starting with `0.1` so this can be considered "experimental"?